### PR TITLE
Enable plots for bench-cpu pipeline

### PR DESCRIPTION
**Pull request by bani-intelaipg**
This PR extends the bench-cpu job by generating csv data and plots from the benchmarks and attaches them as artifacts to the Buildkite CI job.

### CODE REVIEW

File: test/models/tf_ov_model_zoo/gen_plot.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# ==============================================================================
+#  Copyright 2018-2020 Intel Corporation
```
Comment (kanvi-nervana) [inline test/models/tf_ov_model_zoo/gen_plot.py:3]: Change the year to 2021

File: test/models/tf_ov_model_zoo/gen_plot.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--csv',
+        type=str,
+        help="Input CSV file for genearting chart.\n",
```
Comment (adk9) [inline test/models/tf_ov_model_zoo/gen_plot.py:30]: Typo: "generating".

Minor, but you can change occurrences of "chart" in help to "plot".

File: test/models/tf_ov_model_zoo/gen_plot.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    xaxis = np.arange(len(xdata))
+    bar_tf = plt.bar(
+        xaxis,
+        benchmark_data['Stock-TF'],
+        width=width,
+        color='lightgray',
+        label="Stock-TF")
+    bar_stockov = plt.bar(
+        x=xaxis + width,
+        height=benchmark_data['Stock-OV'],
```
Comment (adk9) [inline test/models/tf_ov_model_zoo/gen_plot.py:72]: Minor naming nit, but can we have Stock-TF, OV, TF-OV across the board? There's only one OV so we don't need the "Stock-" qualifier for it, and TFOV should be TF-OV just to be consistent.

File: test/models/tf_ov_model_zoo/run_infer_multiple.sh [hunk 4]
```diff
@@ -24,11 +24,32 @@ while getopts “:m:b:d:h” opt; do
...
+  if [ $# -gt 1 ]; then
+    LINK=$(printf "$LINK;content='%s'" "$2")
+  fi
+  printf '\033]1339;%s\a\n' "$LINK"
+}
+
+# Display image in BuildKite CI Web
+function inline_image {
+  printf '\033]1338;url='"$1"';alt='"$2"'\a\n'
+}
```
Comment (adk9) [inline test/models/tf_ov_model_zoo/run_infer_multiple.sh:39]: This seems to be used. Should we display the plots inline and have links to the CSV in the log output?

File: test/models/tf_ov_model_zoo/run_infer_multiple.sh [hunk 5]
```diff
@@ -24,11 +24,32 @@ while getopts “:m:b:d:h” opt; do
...
+#==============================================================================
+#==============================================================================
+
 [ -f "$MANIFEST" ] || ( echo "Manifest not found: $MANIFEST !"; exit 1 )
 MANIFEST="$(cd "$(dirname "$MANIFEST")"; pwd)/$(basename "$MANIFEST")" # absolute path
 
 cd ${WORKDIR} || ( echo "Not found: $WORKDIR !"; exit 1 )
 echo "Dir: ${WORKDIR}"
+CSVFILE=benchmark_avg_infer_msec.csv
+[ -f "$CSVFILE" ] && rm $CSVFILE
```
Comment (adk9) [inline test/models/tf_ov_model_zoo/run_infer_multiple.sh:50]: Typically "rm -f $CSVFILE" works for when you want to delete a file if it exists.

File: test/models/tf_ov_model_zoo/run_infer_multiple.sh [hunk 6]
```diff
@@ -46,6 +67,18 @@ while read -r line; do
     (( $retcode == 1 )) && failed_models+=("${args[0]}")
 done < "$MANIFEST"
 
+if [ "$BENCHMARK" == "YES" ] && [ -f "$CSVFILE" ]; then
```
Comment (adk9) [inline test/models/tf_ov_model_zoo/run_infer_multiple.sh:70]: Not directly related to this PR, but in terms of ease-of-use, "-b" would be better for benchmark mode than an unqualified "YES" or "NO".

File: test/models/tf_ov_model_zoo/gen_plot.py (hunk 3 above)
Comment (bani-intelaipg) [reply test/models/tf_ov_model_zoo/gen_plot.py:72]: Ok.

File: test/models/tf_ov_model_zoo/run_infer_multiple.sh (hunk 4 above)
Comment (bani-intelaipg) [reply test/models/tf_ov_model_zoo/run_infer_multiple.sh:39]: Deleting unused funcs. Still having path/location issues to display inline image

File: test/models/tf_ov_model_zoo/run_infer_multiple.sh (hunk 6 above)
Comment (bani-intelaipg) [reply test/models/tf_ov_model_zoo/run_infer_multiple.sh:70]: Yes, but keeping the argument parsing simple in .sh script with mandatory value: -b YES

File: test/models/tf_ov_model_zoo/run_infer_multiple.sh [hunk 7]
```diff
@@ -24,11 +24,18 @@ while getopts “:m:b:d:h” opt; do
...
+#==============================================================================
+#==============================================================================
+
 [ -f "$MANIFEST" ] || ( echo "Manifest not found: $MANIFEST !"; exit 1 )
 MANIFEST="$(cd "$(dirname "$MANIFEST")"; pwd)/$(basename "$MANIFEST")" # absolute path
 
 cd ${WORKDIR} || ( echo "Not found: $WORKDIR !"; exit 1 )
 echo "Dir: ${WORKDIR}"
+CSVFILE=benchmark_avg_infer_msec.csv
+rm $CSVFILE
```
Comment (adk9) [inline test/models/tf_ov_model_zoo/run_infer_multiple.sh:36]: This will error out if the file doesn't exist.

Comment (bani-intelaipg) [reply test/models/tf_ov_model_zoo/run_infer_multiple.sh:36]: reverted

**Review by adk9 — APPROVED** [review]

**Pull request merged**